SimulationAgent/Agent.cs

```diff
@@ -59,17 +63,27 @@ public async Task RunAsync()
                 {
                     this.log.Debug("------ Checking for simulation changes ------");
 
-                    var newSimulation = (await this.simulations.GetListAsync()).FirstOrDefault();
-                    this.log.Debug("Simulation loaded", () => new { newSimulation });
+                    var simulationList = await this.simulations.GetListAsync();
+
+                    // currently we support only 1 running simulation so the result should return only 1 item
```

Comment:
please add to the comment: "The list of simulations returned is sorted by creation date, so the agent runs the first that was created"